requirements/base.txt

```diff
@@ -8,55 +8,56 @@ asgiref==3.8.1
     # via django
 backports-zoneinfo==0.2.1 ; python_version < "3.9"
     # via
-    #   -c requirements/constraints.txt
+    #   -c /home/runner/work/edx-enterprise-subsidy-client/edx-enterprise-subsidy-client/requirements/constraints.txt
```

Comment:
We need to include global constraints in the `pip.in` file as well to pin the `pip` version and in-turn avoid this long path from occurring.